site/content/docs/getting-further/is-my-app-testable.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Is My App Security Testable?"
+type: page
+tags: 
+- advanced
+EditableContent: true
+addBreadcrumbs: true
+---
+Web app security is hard, and there are many, many aspects to it (see [OWASP](https://www.owasp.org) for more details).
+
+When it comes to security testing the best way to test the security of a web app is via a pentest conducted by experts.
+However such pentests are expensive, and I have not heard of any company that can afford to carry out
+regular pentests on all of their web apps.
+
+This is where ZAP can shine. ZAP is a cheap and effective way to test all of your web apps, whenever you need them tested.
+
+ZAP can help you secure your web apps, but you can make life much more difficult for ZAP, and that in turn makes life more difficult for you.
+
+Making it harder for ZAP to test your web apps will not make them more secure, it can in fact make them less secure.
+If you make automated testing too hard then the race to find vulnerabilities in your apps will be between
+the pentesters you hire and the malicious attackers who are out to compromise you.
+
+### Provisioning
+
+Can you set up your app in a test environment with no manual input?
+
+This includes provisioning all of the services, configuring them to work together, and creating a good range of test data, and provisioning test users for all valid roles.
+Your app should not depend on any third party services that cannot be mocked, or interact with any other services which are not part of the isolated test environment.
+
+If you cannot set easily set up your app in a reliable state in a dedicated test environment then the reliablility of the results you get will vary.
+
+### Performance
+
+If your test environment is slow under normal circumstances then it will really struggle when scanned by ZAP.
```

Comment:
"when scanned." period, no use mentioning ZAP, it isn't like it's unique or particularly hard on things.

Reply:
I still feel like this "targets" ZAP by mentioning it. But it's slightly better than before.